src/patternfly/utilities/Sizing/sizing.hbs

```diff
@@ -0,0 +1,6 @@
+<div class="item{{#if sizing--modifier}} {{sizing--modifier}}{{/if}}"
+  {{#if sizing--attribute}}
+    {{{sizing--attribute}}}
+	{{/if}}>
```

Comment:
Can you fix the indentation in this file?

Reply:
Must've been some weird newline stuff that VS Code was correcting automatically for me. Fixed.